standalone/src/models/product-selection/product-of-selection/fields-config.ts

```diff
@@ -0,0 +1,14 @@
+import { type TModelFieldsConfig } from '@/core';
+import { ReferenceGraphql } from '@/models/commons';
+import type { TProductOfSelectionGraphql } from './types';
```

Comment:
very nit but the `type` placement in the imports are inconsistent

Reply:
Sorry, what do you mean?

Reply:
Ah, I see. I think these particular imports were generated by CLI, so we might want to fix it there.

Reply:
The CLI does not know if the implementation of the file will require to import anything else from the `core` package so that's why it uses the safe way of importing a type.
Actually. the `type` keyword is optional and only used for some optimizations the bundler can do which I'm not sure they even apply in our case.

Please, focus on the relevant parts of the code 🙏

Reply:
sorry still trying to figure out the relevant parts 🫡